scripts/generate-global-changelog.js

```diff
@@ -95,7 +101,9 @@ function main() {
 
 	const files = diffOutput.split('\0').filter(Boolean);
 
-	let output = `# CHANGELOG\n## ${releaseDateString}\n`;
+	let outputChanges = `# CHANGELOG\n## ${releaseDateString}\n`;
+
+	let versionBumps = `### Packages with a version bump only\n`;
```

Comment:
Probably never going to happening, but if there are ever any times when we _don't_ have a version bump, this would print an empty heading. Even though it's unlikely, it's good to handle "null state" cases, so that could be something like:

```js
const versionBumps = [];

// ...

if (bumpOnly) {
    versionBumps.push(...);
}

// ...

if (versionBumps.length) {
    outputChanges += '\n### Packages with a version bump only\n' +
        versionBumps.join('\n\n');
}
```

or something similar.